<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPlotsViewer</class>
 <widget class="QWidget" name="CPlotsViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CPlotsViewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../DyssolGUI/Resources.qrc">
    <normaloff>:/Icons/WinIcon</normaloff>:/Icons/WinIcon</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximum">
        <number>0</number>
       </property>
       <property name="tracking">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      <property name="toolTip"><string>Navigate through all curves in the selected plot</string></property><property name="whatsThis"><string>Navigate through all curves in the selected plot</string></property></widget>
     </item>
     <item>
      <widget class="QLabel" name="labelZName">
       <property name="text">
        <string>Z param</string>
       </property>
      <property name="toolTip"><string>Current plot parameter value</string></property><property name="whatsThis"><string>Current plot parameter value</string></property></widget>
     </item>
     <item>
      <widget class="QLabel" name="labelZValue">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      <property name="toolTip"><string>Current plot parameter value</string></property><property name="whatsThis"><string>Current plot parameter value</string></property></widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>-1</number>
     </property>
    <property name="toolTip"><string>Simulation results</string></property><property name="whatsThis"><string>Simulation results</string></property></widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>horizontalSlider</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources>
  <include location="../../DyssolGUI/Resources.qrc" />
 </resources>
 <connections />
</ui>
